src/components/pre-footer/data.ts

```diff
@@ -37,38 +37,38 @@ const top_adopters = {
       name: "WeScale",
       image: "/svg/weScale.png",
     },
-    {
-      name: "Kublr",
-      image: "/svg/kublr.png",
-    },
+    // {
+    //   name: "Kublr",
+    //   image: "/svg/kublr.png",
+    // },
     {
       name: "Zebrium",
       image: "/svg/Zebrium.svg",
     },
-    {
-      name: "Orange",
-      image: "/svg/Orange.jpeg",
-    },
-    {
-      name: "Kube Flare",
-      image: "/svg/KubeFlare.png",
-    },
-    {
-      name: "Anuta Networks",
-      image: "/svg/AnutaNetworks.png",
-    },
-    {
-      name: "Kitopi",
-      image: "/svg/Kitopi.png",
-    },
-    {
-      name: "Lenskart",
-      image: "/svg/lenskart.svg",
-    },
-    {
-      name: "Halodoc",
-      image: "/svg/halodoc.svg",
-    }
+    // {
+    //   name: "Orange",
+    //   image: "/svg/Orange.jpeg",
```

Comment:
remove commented code if not needed

Reply:
We might need to add adopters to show up, or maybe we'll add a infinite scroll with the adopter logos so let's keep this